.kitchen.vagrant.yml

```diff
@@ -0,0 +1,101 @@
+---
+driver:
+  name: vagrant
+
+provisioner:
+  name: ansible_playbook
+  hosts: localhost
+  roles_path: ../
+  require_ansible_repo: false
+  require_ansible_omnibus: false
+  require_ansible_source: false
+  require_pip: true
+  ansible_version: 2.4.3.0
+  http_proxy: <%= ENV['HTTP_PROXY'] %>
+  https_proxy: <%= ENV['HTTPS_PROXY'] %>
+  no_proxy: localhost,127.0.0.1
+  ignore_extensions_from_root: [".git",".idea",".kitchen.yml"]
+  ignore_paths_from_root: [".git",".idea",".kitchen"]
+  <% if ENV['VERSION'] %>
+  attributes:
+    extra_vars:
+      es_major_version: "<%= ENV['VERSION'] %>"
+      <% if ENV['VERSION'] == '5.x' %>
+      es_version: '5.6.7'
+      <% end %>
+  <% end %>
+
+verifier:
+  name: inspec
+  sudo: true
+  inspec_tests:
+    - git: https://github.com/elastic/elasticsearch-inspec.git
+      branch: 6.0
```

Comment:
Is there a way to pin this to a specific release/version? If this testing becomes part of the automated pull request testing I wouldn't want the tests to randomly fail just because the inspec tests had been updated.

Reply:
We had some conversations on our end about that. We found that there were some significant changes between the 5.x and 6.x versions of elastic. At the time the inspec profile was built it was during 5.x so some controls were failing for me (I was on 6.1). 

The two camps of thought was either keep it one repo and make a branch for each major version or make separate specific repos for each version. I went with the branch way because I thought it would be "cleaner", but it can be changed too.

Reply:
That makes a lot of sense. However I was actually referring to version of the elasticsearch-inspec project itself. If these tests become part of the automated testing pipeline for pull-requests I would want to have the version pinned to a specific git commit hash or version so that someone pushing changes to the elasticsearch-inspec repo wouldn't cause all pull requests to start failing in ansible-elasticsearch.